public/conduct.jade

```diff
@@ -0,0 +1,30 @@
+div.jsla-toplevel-container.jsla-content-page
```

Comment:
By the looks of the diff, it seems that this is a new file and not a "moved" file. If it was moved you normally see, `old name ---> new name`.

Maybe I'm just misreading this. I was just thinking we could probably do something like what we do for the `js.la/youtube` route,
```jade
script.
    window.location = 'http://js.la/code-of-conduct'
```